Gemfile

```diff
@@ -156,10 +156,8 @@ group :test, :development do
   gem 'rails_best_practices', "~>1.15.4"
   gem 'simplecov', '~> 0.7.1', :require => false
 
-end
-
-group :development do
   gem 'pry'
```

Comment:
debugging sounds like a thing that belongs to `development`.
Debugging should not be in `testing`.